src/components/Container.tsx

```diff
@@ -86,14 +86,14 @@ export default function () {
   };
 
   const previous = () => {
-    if(onPrevious != undefined){
+    if (onPrevious != undefined) {
       onPrevious();
     }
     setCurrentIdWrapper((prev) => (prev > 0 ? prev - 1 : prev));
   };
 
-  const next = () => {
-    if(onNext != undefined){
+  const next = (isSkippedByUser?: boolean) => {
```

Comment:
This method looks better! Would also suggest the following change to make the argument more expressive:
```suggestion
  const next = (options?: { isSkippedByUser?: boolean }) => {
```